Tools/GNUMake/Make.defs

```diff
@@ -111,6 +111,12 @@ ifeq ($(USE_PMI), TRUE)
   DEFINES += -DAMREX_PMI
 endif
 
+ifeq ($(USE_OMP_OFFLOAD), TRUE)
+  DEFINES += -DOMP_OFFLOAD
+  INCLUDE_LOCATIONS += $(CUDA_DIR)/include
```

Comment:
There is some logic down below that should cover this if MAKE_CUDA_PATH is set.

Reply:
Ah OK, I deleted these and it works, thanks.